.github/workflows/publish-docs.yml

```diff
@@ -21,26 +21,31 @@ jobs:
         with:
           python-version: "3.12"
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
+
       - name: Create and activate virtual environment
         run: |
-          python -m venv venv
+          uv venv venv
           source venv/bin/activate
 
       - name: Install pdoc and other dependencies
         run: |
           source venv/bin/activate
-          python -m pip install pdoc3 setuptools
+          uv install pdoc3 setuptools
 
       - name: Install package
         run: |
           source venv/bin/activate
-          pip install ./livekit-agents
-          ./livekit-plugins/install_local.sh
```

Comment:
NOTE: I can't find this script in the git history, unclear what it was for?

Reply:
this has been removed from v1.. 

actually for this docs workflow.. we'd need to think about v1 vs pre-v1.. do we want to keep separate reference docs?

Reply:
There's probably something clever we can do using the branch name itself in the deploy script to upload to two different S3 buckets. Since we only run this manually, we could say `dev-1.0` (eventually promoted to `main`) pushes to the current bucket, and `0.X` or whatever `main` is demoted to is deployed to a second bucket?

Reply:
maybe it's fine.. let's move forward then.